Makefile-az.mk

```diff
@@ -315,3 +269,18 @@ az-pprof-enable: ## Enable profiling
 az-pprof: ## Profile
 	kubectl port-forward service/karpenter -n karpenter 8000 &
 	sleep 2 && go tool pprof -http 0.0.0.0:9000 localhost:8000/debug/pprof/heap
+
+az-mkaks-user: az-mkrg ## Create compatible AKS cluster, the way we tell users to
+	hack/deploy/create-cluster.sh $(AZURE_CLUSTER_NAME) $(AZURE_RESOURCE_GROUP)
```

Comment:
nit: after running this, as it stalling during that I believe was cluster creation, I think it'd be nice to have a few more statements about what the steps it completing are posted to the command line.

Reply:
Added